hrui/hrui.php

```diff
@@ -418,28 +418,47 @@ function hrui_civicrm_upgrade($op, CRM_Queue_Queue $queue = NULL) {
  * Implementation of hook_civicrm_tabs
  */
 function hrui_civicrm_tabs(&$tabs, $contactID) {
-  $newTabs = array();
-
   foreach ($tabs as $i => $tab) {
-    if ($tab['id'] != 'log') {
-      $newTabs[$i] = $tab['title'];
-    }
-    else {
-      $changeLogTabID = $i;
+    switch($tab['title'])  {
+      case 'Assignments':
+        $tabs[$i]['weight'] = 20;
+        break;
+      case 'Identification':
+        $tabs[$i]['weight'] = 50;
```

Comment:
Maybe we should add a comment here explaining why the weight jumps from 20 to 50 (because of tasks and assignments). Or even better, document the method explaining the tab order and that it depends on other extensions.